src/components/userAuth/signInMail/NewPassword.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, Button, TextField, Typography } from '@mui/material'
+import Logo from 'components/logo/Logo'
+import { commonStyles } from 'styles/commonStyles'
+import { makeStyles } from 'tss-react/mui'
+import { useForm } from 'react-hook-form'
+import { useState } from 'react'
+
+type FormData = {
+  email: string
+}
+
+const NewPassword = () => {
+  const commonClasses = commonStyles().classes
+  const { classes } = useStyles()
+  const {
+    register,
+    watch,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormData>({
+    defaultValues: {
+      email: '',
+    },
+  })
+
+  const inputEmail = watch('email')
+  const [success, setSuccess] = useState(false)
+
+  const onSubmit = () => {
+    setSuccess(true)
+  }
+  return (
+    <Box className={commonClasses.mainBox}>
+      <Logo />
+      {!success ? (
```

Comment:
Maybe split to separate components, bc it's hard to read ternary condition here

Reply:
I agree. Done.